### ISSUE TITLE
machinery build fails if repositories are cd repositories

### ISSUE THREAD

**Issue by tgoettlicher** [SUSE/machinery#617]
If only the system description only contains cd repositories `machinery build` fails with:
```
Feb-18 14:30:05 <3> : unknown-50860d0:0: Relax-NG validity error : Expecting an element users, got nothing
unknown-50860d0:0: Relax-NG validity error : Expecting an element repository, got nothing
unknown-50860d0:0: Relax-NG validity error : Invalid sequence in interleave
unknown-50860d0:0: Relax-NG validity error : Element image failed to validate content
```

**Comment by tgoettlicher** [comment]
When fixing this bug, please add a message which describes that you need an accessible online repository. More and more users experience trouble when building images caused by repository configuration.

**Comment by thardeck** [comment]
Should be fixed by #625
